colorDC1/Dockerfile

```diff
@@ -128,13 +126,18 @@ RUN echo "gea_user ALL=(ALL) NOPASSWD: ALL" >> /etc/sudoers
 # Switch to GEA user
 COPY ./jupyter_notebook_config.json /opt/conda/etc/jupyter/jupyter_notebook_config.json
 COPY ./jupyter_notebook_config.py /home/gea_user/.jupyter/jupyter_notebook_config.py
-COPY ./.bashrc /home/gea_user/.bashrc
-RUN chown -R gea_user:gea_user /home/gea_user/.jupyter
-RUN chmod -R 777 /home/jovyan
-RUN chown gea_user:gea_user /home/gea_user/.bashrc
+COPY ./_bashrc /home/gea_user/.bashrc
+COPY ./_bash_prompt /home/gea_user/.bash_prompt
+RUN mkdir /home/gea_user/work && \
+    chown -R gea_user:gea_user /home/gea_user
+WORKDIR /home/gea_user/work
```

Comment:
`/home/gea_user/work` is intended to be the new working directory, and it should be empty in the image. The DE will mount this directory from the file system of the node where the analysis is running, so any files that are in this directory in the image itself will be masked. This also means that this directory should be specified as the working directory when you're defining the tool in the DE:

![image](https://user-images.githubusercontent.com/415143/90678956-4d295880-e214-11ea-93a0-80bc926fe04e.png)